src/ui/components/RecommendedBadge/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import '~ui/css/styles';
+
+$recommended-text-color: #e25920;
```

Comment:
Is this part of Photon? https://design.firefox.com/photon/visuals/color.html

If not, shouldn't we be using a Photon color here?

Reply:
I added a comment about this to the diff when submitting this PR. The color specified in the InVision markup doesn't exist in `photon-colors`. There is an [orange color for Photon](https://design.firefox.com/photon/visuals/color.html#orange), but the style guide specifically states "Firefox Orange is only used for branding. Do not use it otherwise." so I'm not sure we should use that. @brassy- can you please comment on this?

Reply:
I guess we can use variations of [Firefox Orange](https://design.firefox.com/photon/visuals/color.html#orange) for now and await confirmation from @brassy- .